jpype/beans.py

```diff
@@ -89,10 +89,10 @@ class _BeansCustomizer(object):
     def __jclass_init__(self):
         accessor_pairs = _extract_accessor_pairs(self.__dict__)
         for attr_name, (getter, setter) in accessor_pairs.items():
-            attr_name = pysafe(attr_name)
+            attr_name = _pysafe(attr_name)
 
             # Don't mess with an existing member
-            if attr_name in self.__dict__:
+            if attr_name is None or attr_name in self.__dict__:
```

Comment:
I think this was missed in [#1056](https://github.com/jpype-project/jpype/pull/1056)